bootstrap/src/main/java/org/jboss/windup/bootstrap/Bootstrap.java

```diff
@@ -258,31 +257,32 @@ else if ("--generateCompletionData".equals(arg))
             furnaceService.getFurnace().addRepository(AddonRepositoryMode.MUTABLE, new File(getUserWindupDir(), "addons"));
         }
 
+        if (command == null && updateRulesets && onlyRulesetsUpdateRequested(arguments))
```

Comment:
What happens if --updateRulesets is specified in addition to windup execution/analysis information? It currently looks like it will skip the ruleset update silently. That doesn't seem right.